hayeon/baekjoon/수학/10250-ACM-호텔.js

```diff
@@ -0,0 +1,53 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  //테스트 케이스
+  const testCase = +input[0];
+  let result = "";
+
+  for (let i = 1; i <= testCase; i++) {
+    const [h, w, n] = input[i].split(" ").map(Number);
+    let floor = n % h;
+    let room = Math.ceil(n / h);
+
+    if (floor === 0) {
+      floor = h;
+    }
+
+    room < 10 ? (room = `0${room}`) : room;
+
+    result += `${floor}${room}\n`;
+  }
+
+  console.log(result);
```

Comment:
```suggestion
  //테스트 케이스
  const testCase = +input[0];
  let result = "";

  for (let i = 1; i <= testCase; i++) {
    const [h, w, n] = input[i].split(" ").map(Number);
    const floor = n % h === 0 ? h : n % h;
    let room = Math.ceil(n / h);

    room = room < 10 ? `0${room}` : room;

    result += `${floor}${room}\n`;
  }

  console.log(result);
```
- 풀이잘봤습니다 🙇‍♂️
- 약간의 교정만 해드렸씁니다 고생하셨씁니다.

Reply:
감사합니다!! 이렇게도 풀 수 있었군뇨....👍🏻